macstadium-pod-2/main.tf

```diff
@@ -453,8 +455,8 @@ module "collectd-vsphere-common" {
   librato_token                           = "${var.librato_token}"
   env                                     = "common"
   index                                   = "${var.index}"
-  collectd_vsphere_collectd_network_user  = "${var.collectd_vsphere_collectd_network_user}"
-  collectd_vsphere_collectd_network_token = "${var.collectd_vsphere_collectd_network_token}"
+  collectd_vsphere_collectd_network_user  = "collectd-vsphere-2"
```

Comment:
var name needs to be declared since we're generating here, right?

Reply:
⬆️ @Lyoness I don't follow 😁   Help!